terra/tools/prepare-token.ts

```diff
@@ -0,0 +1,25 @@
+import { deploy_contract, instantiate_contract, query_contract } from './utils';
+
+async function script() {
+    const TEST_ADDRESS: string = 'terra1x46rqay4d3cssq8gxxvqz8xt6nwlz4td20k38v';
+    let code_id = await deploy_contract('../artifacts/cw20_base.wasm');
```

Comment:
Is the wasm code deterministically built? If so, fine to commit to repo, otherwise not